README.md

```diff
@@ -36,14 +36,14 @@ There are two main modules, `akka-stream-json` and `akka-http-json`.
 
 ```
 libraryDependencies ++= List(
-  "org.mdedetrich" %% "akka-stream-json" % "0.2.0",
-  "org.mdedetrich" %% "akka-http-json" % "0.2.0"
+  "org.mdedetrich" %% "akka-stream-json" % "0.2.1",
```

Comment:
If its a binary compatible change (which it is), the version should be `0.3.1`.

Reply:
sure, have bumped the minor version now instead of patch version, so the new version is now `0.3.0`